docs/How-To-Create-Evergreen-Projects.md

```diff
@@ -0,0 +1,27 @@
+# How to Create Evergreen Projects
+
+This guide is meant to help users create their own projects on Evergreen without creating a ticket and waiting. 
+
+## Authorization
+
+To have access to create a project, the user must be either a project admins or super user.
+If you do not see the `New Project` button on the project settings page, or do not have access to the project settings page, you may request admin permissions from MANA or ask a current admin to add you as one.
+
+## Steps to Create
+
+1. Visit the projects page in the new UI https://spruce.mongodb.com/projects/.
+2. Click New Project. If you want a current project copied, click Duplicate Current Project. Otherwise, click Create New Project.
+3. Enter the project name to match the GitHub repo name. Users can change this later.
```

Comment:
Since this is information for users and not us, we can just say Enter the project name, repo org, and repo name to combine steps 3 and 4.

Reply:
done in wiki